source/plugins/ruby/arc_k8s_cluster_identity.rb

```diff
@@ -34,7 +33,8 @@ def initialize
     @isLastTokenRenewalUpdatePending = false
     @token_file_path = "/var/run/secrets/kubernetes.io/serviceaccount/token"
     @cert_file_path = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-    @kube_api_server_url = KubernetesApiClient.getKubeAPIServerUrl
```

Comment:
Can we remove this function completely from the source file? I dont think we use this anywhere else